### ISSUE TITLE
Unable to keep Settings Storage selected on Android 13

### ISSUE THREAD

**Issue by privacyguy123** [akhilkedia/AllTrans#62]
Is this a problem? Using Lsposed

Some say this is due to the module being developed for Lspatch and not Lsposed, which there are some differences.

**Comment by zinhuofc** [comment]
That's right, the dev uses lspatch, so to go out to lsposed he said he was going to use an emulator, but gave no news if it worked or if he is working
the last version took time, I was so happy when it launched because it is my favorite app, until I was sad again since it does not work in lsposed, and for my use lspatch does not work well in multiuser

**Comment by privacyguy123** [comment]
Unfortunately dev doesn't seem to care. Perhaps a donation would get things moving?

**Comment by zinhuofc** [comment]
> Unfortunately dev doesn't seem to care. Perhaps a donation would get things moving?

Maybe who knows, he never said about donations, has offered to help other devs pick up the project, but none have been interested, I don't know if he has little time

**Comment by zinhuofc** [comment]
https://github.com/LSPosed/LSPosed/issues/2732 

I just saw today this problem in addition to Android 13 it is at 10, if you touch the LSPOSED settings the settings will no longer check, 13 never checks, according to the LSPOSED dev the developer has to fix it
tested on 1.9.3 on 10
and 1.9.6 at 13
